fem/gridfunc.hpp

```diff
@@ -671,10 +671,9 @@ class GridFunction : public Vector
    /// Save the GridFunction to an output stream.
    virtual void Save(std::ostream &out) const;
```

Comment:
We could add a `Print` method to mirror the `Mesh` interface, and make `Save(std::ostream&)` call it. I'm just not sure it's not too much :-)

Reply:
I'd prefer to keep it simple for the users and unify to `Save`.

The Save/Print has been pointed out as one of the interface inconsistencies in https://github.com/mfem/mfem/issues/1557